spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -251,8 +251,10 @@ protected void shutdownAndWait(Instance instance) {
 		try {
 			int timeout = getLocalDeployerProperties().getShutdownTimeout();
 			if (timeout > 0) {
+				logger.debug("About to call shutdown endpoint for instance {}", instance);
```

Comment:
I think it would be better: for instance -> for the instance. will fix here and in other places when merging.